packages/feedback/src/SmileField.js

```diff
@@ -35,6 +35,7 @@ const SmileField = ({ name, options, onChange, autoFocusFeedbackButton, onClose,
           onChange(option);
         }
       }}
+      aria-label={option.label}
```

Comment:
I'm not sure that each `option` is guaranteed to a have a `label`, so providing a unique default like `aria-label={ option.label || feedbackButton${i} }` may be good here.

We also want the `FeedbackButton` itself to have the `aria-label` prop on the `Button` it renders. This happens by chance now because we spread `...rest` there, but this only makes `FeedbackButton` more accessible if you use it through `SmileField`

Current code:
```js
const FeedbackButton = ({ onClick, icon, active, children, iconSize, ...rest }) => (
  <Button
    style={btnStyles}
    color={active === icon ? 'primary' : 'light'}
    aria-pressed={active === icon}
    onClick={() => {
      onClick(icon);
    }}
    onKeyDown={({ keyCode }) => keyCode === 13 && onClick(icon)}
    {...rest} // aria-label from SmileField gets spread here
  >
    <Icon data-testid="feedback-icon" name={icon} size={iconSize} style={iconStyles} />
    <span className="sr-only">{children}</span>
  </Button>
);
```

But maybe we can do something like:
```js
const FeedbackButton = ({ onClick, icon, active, children, iconSize, ariaLabel, ...rest }) => (
  <Button
    style={btnStyles}
    color={active === icon ? 'primary' : 'light'}
    aria-pressed={active === icon}
    onClick={() => {
      onClick(icon);
    }}
    onKeyDown={({ keyCode }) => keyCode === 13 && onClick(icon)}
    aria-label={ariaLabel || someDefault}
    {...rest}
  >
    <Icon data-testid="feedback-icon" name={icon} size={iconSize} style={iconStyles} />
    <span className="sr-only">{children}</span>
  </Button>
);
```

and have `SmileField` pass in an `ariaLabel` prop. What do you think?

Reply:
Why are we adding an aria-label when there's already an accessible name via the sr-only span? we don't want competing labels.

Reply:
Ah good call, I didn't even notice that 😄